app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -274,7 +287,17 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         if(applicationKvStore.getBoolean("login_skipped")){
             delete.setVisibility(GONE);
         }
-      
+        /**
+         * gets the height of the frame layout as soon as the view is ready and updates aspect ratio
```

Comment:
Just a minor thing, we start Javadocs with a capital letter. Could you please arrange the rest of them accordingly?

Reply:
Done.